src/overflowMenu/__tests__/OverflowMenu.test.js

```diff
@@ -99,4 +99,14 @@ describe('overflowMenu', () => {
       });
     }
   );
+
+  it('should not render overflow button when no item is specified', () => {
+    const onPress = jest.fn();
+
+    const { queryByA11yLabel } = render(
+      <OverflowMenu OverflowIcon={<Text>+</Text>} onPress={onPress} />
```

Comment:
I think we also want to handle the case when we have

```
      <OverflowMenu OverflowIcon={<Text>+</Text>} onPress={onPress}>
        {false && <HiddenItem icon={<Text>O</Text>} title="search" onPress={...} />}
      </OverflowMenu>
```

and show nothing in that case

Reply:
Thats actually pretty good point. Updated the PR to reflect this.